ios/purchases_flutter.podspec

```diff
@@ -20,7 +20,11 @@ Pod::Spec.new do |s|
   s.swift_version         = '5.0'
 
   # Flutter.framework does not contain a i386 slice.
-  s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES', 'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386' }
+  s.pod_target_xcconfig = { 
+    'DEFINES_MODULE' => 'YES',
+    'EXCLUDED_ARCHS[sdk=iphonesimulator*]' => 'i386',
+    'OTHER_LDFLAGS' => '-framework Purchases'
```

Comment:
this line is the actual fix. it adds a missing linker flag that links the Purchases framework. 
I'm not sure why this flag isn't added automatically, I'm still figuring that out. But it did fix the linker issues on the test project I'm using